pkg/component/controller/kubeproxy.go

```diff
@@ -111,6 +112,34 @@ func (k *KubeProxy) getConfig(clusterConfig *v1beta1.ClusterConfig) (proxyConfig
 		DualStack:            clusterConfig.Spec.Network.DualStack.Enabled,
 		Mode:                 clusterConfig.Spec.Network.KubeProxy.Mode,
 		MetricsBindAddress:   clusterConfig.Spec.Network.KubeProxy.MetricsBindAddress,
+		// Default iptables settings
+		IPTables: map[string]interface{}{
+			"masqueradeAll": false,
+			"masqueradeBit": "null",
```

Comment:
`"null"` string feels bit odd here, is there specific reason to use it as string?

Reply:
Well, I need it to be `masqueradeBit: null` in the YAML. If I set it to `nil`, in the final YAML it looks like `masqueradeBit: <no value>`

Reply:
I could suggest make a constant named say `nullString` and use it to avoid such questions once in a while.

```

const nullString = "null"
...
"masqueradeBit": nullString
...
```

Reply:
@mikhail-sakhnov  Good idea, thanks. Done